Jenkinsfile

```diff
@@ -30,8 +31,33 @@ pipeline {
     CI = true
     CC_TEST_REPORTER_ID = '06c1254d7c2ff48f763492791337193c8345ca8740c34263d68adcc449aff732'
   }
+  parameters {
+      string(name: 'TALOS_PACKAGES', defaultValue: '', description: 'Comma seperated list of packages to have talos bump. e.g. "@bbc/psammead-styles,@bbc/psammead-brand"')
+  }
   stages {
+    stage ('Install') {
+      agent {
+        docker {
+          image "${nodeImage}"
+          label nodeName
+          args '-u root -v /etc/pki:/certs'
+        }
+      }
+      steps {
+        sh 'make install'
+      }
+      post {
+        always {
+          script {
+            stageName = env.STAGE_NAME
+          }
+        }
+      }
+    }
     stage ('Run application tests') {
+      when {
+        expression { params.TALOS_PACKAGES == '' }
```

Comment:
Why do you want to skip running tests if Talos is run manually?

Reply:
Just doesn't seem necessary IMO, you would manually invoke it on latest, where its already merged, so dont need to test it again